webapps/frontend/app/views/site/vdisk/vdisks.html

```diff
@@ -16,16 +16,24 @@
 but WITHOUT ANY WARRANTY of any kind.
 -->
 <section>
-    <div>
-        <h2 class="pull-left" data-i18n="ovs:vdisks.title"></h2>
-        <button class="btn btn-default pull-right" id="buttonAddVdisk"
-                data-bind="click: addVDisk, enable: $root.shared.user.roles().contains('manage') && runningVPool()"
-                style="margin-top: 20px; margin-bottom: 10px;">
-            <span class="fa" data-i18n="[html]ovs:icons.add"></span>&nbsp;
-            <span data-i18n="ovs:vdisks.add"></span>
-        </button>
+    <div class="container-fluid">
+        <div class="row row-no-padding">
+            <div class="col-lg-4">
+                <h2 data-i18n="ovs:vdisks.title"></h2>
+            </div>
+            <div class="col-lg-8">
+                <div class="pull-right h2">
+                    <div data-bind="searchbar: { query: vdisk_search, defaultField: 'name', placeholder: 'Search vdisks', defaultQuery: query}" style="display: inline-block;"></div>
```

Comment:
Placeholder strings should be defined in ovs.json and use `vDisks` instead of `vdisks` as we do everywhere in the GUI